Header header
bool charging 
float64 battery1_voltage
float64 battery2_voltage